pyannote/core/feature.py

```diff
@@ -235,8 +235,12 @@ def crop(
         return SlidingWindowFeature(data, sliding_window, labels=self.labels)
 
     def _repr_png_(self):
-        from .notebook import repr_feature
+        from .notebook import MATPLOTLIB_IS_AVAILABLE, MATPLOTLIB_WARNING
+        if not MATPLOTLIB_IS_AVAILABLE:
+            warnings.warn(MATPLOTLIB_WARNING.format(obj=self))
```

Comment:
Shouldn't `obj` be the class name rather than the instance?

```python
>>> MATPLOTLIB_WARNING.format(obj=Annotation())
Couldn't import matplotlib to render the vizualization for object . To enable, install the required dependencies with 'pip install pyannore.core[notebook]'

>>> MATPLOTLIB_WARNING.format(obj=Annotation().__class__.__name__)
Couldn't import matplotlib to render the vizualization for object Annotation. To enable, install the required dependencies with 'pip install pyannore.core[notebook]'
```

Reply:
Good point, sorry for that.